src/utils/curve/curve.js

```diff
@@ -27,3 +27,58 @@ export function retrieve({ AMId, businessDate, assetIds }, callback) {
   promise.catch(error => callback(error))
 }
 
+export function insert({ AMID, businessDate }, callback) {
```

Comment:
Where is the actual data to insert?

Reply:
Also, `AMID` is inconsistent with the type file which specifies `AMId`. It should be the latter.